module_1/src/com/epam/cycles/CyclesTaskF.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.cycles;
+import java.util.Scanner;
+
+public class CyclesTaskF {
+
+    public static void main(String[] args) {
+
+        Scanner console = new Scanner(System.in);
+        System.out.println("Set the interval [m,n] including natural numbers: ");
+        System.out.print("-- enter the value 'm': ");
+        int m = console.nextInt();
+        System.out.print("-- enter the value 'n': ");
+        int n = console.nextInt();
+
+        if (m < n && n >= 1) {
```

Comment:
m also should be bigger than 0
```suggestion
        if (m > 0 && n > 1 && m < n) {
```

Reply:
I've kept open the possibility to enter a negative range due to the task conditions aren't limited to input of only natural numbers

Reply:
> I've kept open the possibility to enter a negative range due to the task conditions aren't limited to input of only natural numbers

![image](https://user-images.githubusercontent.com/33148718/96762271-03aae480-13e2-11eb-8940-e0d01db4a1d8.png)